rdrf/rdrf/views/form_view.py

```diff
@@ -1080,6 +1079,37 @@ def _get_template(self):
         return "rdrf_cdes/form_print.html"
 
 
+class FormListView(TemplateView):
+    template_name = "rdrf_cdes/form_list.html"
+
+    @login_required_method
+    def get(self, request, **kwargs):
+        return super().get(request, **kwargs)
```

Comment:
I believe this needs to add security checks, as in [FormView.get](https://github.com/eresearchqut/trrf/blob/f7bc2714c06b574d54b35abe47847440ea665b30/rdrf/rdrf/views/form_view.py#L362-L481)

I don't want to just copy those checks as they are very messy and I'm not sure how useful they all are, but writing them myself could be just as bad if I miss something. 

What things should I be checking for in this method?

Reply:
I'll add: ` security_check_user_patient`